### DISCUSSION TITLE
Where are the default models stored on mac?

### DISCUSSION THREAD

**Discussion by Rentetier** [coqui-ai/TTS#2760] [General Q&A] [answered]
I'm able to run TTS with a default model, for example: 
`$ tts --text "Text for TTS" --model_name "tts_models/en/ljspeech/glow-tts" --out_path output/path/speech.wav`

Then it downloads the model and generates the .wav file. I can also check a list of already downloaded models with `! tts --list_models`:

> Name format: type/language/dataset/model
> 1: tts_models/multilingual/multi-dataset/your_tts [already downloaded]
> 2: tts_models/multilingual/multi-dataset/bark [already downloaded]
> 3: tts_models/bg/cv/vits
> ...

**My question is, where are these models stored on mac, and can I influence the download location of these models?** I could not find the models on my mac and also could not find a default download location.

**Comment by erogol** [comment, marked as answer]
/Users/`user`/Library/Application Support/tts/

**Reply by Rentetier** [reply]
Thanks! :-)
